x/emissions/types/errors.go

```diff
@@ -1,14 +1,13 @@
 package types
 
-// DONTCOVER
+import errorsmod "cosmossdk.io/errors"
 
-import (
-	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
-)
+// DONTCOVER
```

Comment:
```suggestion
```
Already added in codecov.yml that we don't cover this file

Reply:
f2d4d84768a74f837eece318f4b2c9fa557c3f81